syntax_and_semantics/type_restrictions.md

```diff
@@ -69,6 +69,15 @@ restricted_add Six.new, 10
 
 Refer to the [type grammar](type_grammar.html) for the notation used in type restrictions.
 
+Note that type restrictions do not apply to the variables inside the actual methods. This lets you do useful things with them.
```

Comment:
>This lets you do useful things with them

It's a bit wishy washy.

Reply:
How about: `This lets you use the same variable name for different types`?

Reply:
Just leave that "useful" sentence out.

Reply:
Done